docs/mocksupport.rst

```diff
@@ -0,0 +1,131 @@
+.. _`Mock support`:
+
+Mock support
+============
+
+The pywbem PyPI package provides unit testing support for its users via its
+``pywbem_mock`` Python package. This package allows users of pywbem to
+define FakedWBEMConnections and CIM objects (classes, qualifier declarations,
+and instances) that will be processed by these connections. In effect this
+creates a fake WBEM Server with limited capabilities.
+
+The faked HMC environment is set up by creating an instance of the
+:class:`pywebm_mock.FakeWBEMConnection` class instead of the
+:class:`pywbem.WBEMConnection` class:
+
+The fake WBEMConnection is set up by creating an instance of the
+:class:`pywbem_mock.WBEMConnection` class instead of the
+:class:`pywbem_mock.WBEMConnection` class:
+
+.. code-block:: python
+
+    import pywbem
+    import pywbem_mock
+
+    conn = pywbem_mock.FakedWBEMConnection()
+
+    ... Add server CIM object resources (in this case some classes)
```

Comment:
Show the code to add classes.

Reply:
TODO

Reply:
DISCUSS

Reply:
Now shown in list of discussion points, below.